portal/server/controllers/utilisation-report-service/utilisation-report-upload/index.js

```diff
@@ -28,6 +29,33 @@ const setSessionUtilisationReport = (req, nextDueReportDate) => {
   };
 };
 
+/**
+ * @typedef {Object} ReportDetails
+ * @property {number} index - The index of the object
+ * @property {string} error - The error message of the object
+ * @property {string} uploadedByFullName - The index of the object with format '{firstname} {surname}'
+ * @property {string} formattedDateAndTime - The date uploaded formatted as 'h:mmaaa'
+ * @property {string} lastUploadedReportPeriod - The report period of the report formatted as 'MMMM yyyy'
```

Comment:
These properties don't seem to match what we're returning in the `getLastUploadedReportDetails` method.

Reply:
Fixed